pkg/specs/apptype/determine_type.go

```diff
@@ -137,10 +140,9 @@ func (r *inspector) determineTypeFromContents(
 		if err != nil {
 			return "", "", errors.Wrapf(err, "check for %s", filename)
 		}
-	}
-
-	if isReplicatedApp {
-		return "inline.replicated.app", finalPath, nil
+		if isReplicatedApp {
```

Comment:
moved this up because it checks for 2 different files, if the first is true and the second is false... 😢